pkg/services/accesscontrol/roles.go

```diff
@@ -0,0 +1,113 @@
+package accesscontrol
+
+// PredefinedRoles provides a map of permission sets/roles which can be
+// assigned to a set of users. When adding a new resource protected by
+// Grafana access control the default permissions should be added to a
+// new predefined role in this set so that users can access the new
+// resource. PredefinedRoleGrants lists which organization roles are
+// assigned which predefined roles in this list.
+var PredefinedRoles = map[string]RoleDTO{
+	// TODO: Add support for inheritance between the predefined roles to
+	// make the admin ⊃ editor ⊃ viewer property hold.
+	usersAdminRead: {
```

Comment:
This is more related to the frontend and maybe outside this scope, but just want to mention. In the old authorization model we render side menu based on built-in roles. Now we have to support fine grain access control and in order to properly render the menu, we should define roles for each menu item, like `Server Admin -> Users`, `Server Admin -> Orgs`, `Server Admin -> Settings`, etc. That's related to code here

https://github.com/grafana/grafana/blob/b0470c84a5eac8b7871e5a2e4711cadc5e67f93d/pkg/api/index.go#L301-L307

Reply:
Yes, I'm working on that in grafana/grafana-enterprise#1251, but it's out of scope for this PR which "just" covers the API (to do that, I _think_ we're also going to have to change the middlewares for the frontend endpoints) 🙂

Reply:
So in order to make grafana UI behave properly, we have to add that permission. Here or in the other PR.